python/dgl/heterograph.py

```diff
@@ -715,7 +715,10 @@ def dstnodes(self):
     def ndata(self):
         """Return the data view of all the nodes.
 
-        **Only works if the graph has one node type.**
+        If the graph has only one node type. The g.ndata directly represents the
+        node data.
+        If the graph has multiple node type. The g.ndata is a dictionary of node
+        types and their corresponding node data.
```

Comment:
How about "If the graph has multiple node types, then g.ndata[feat][node_type]" gives the features associated with key `feat` for `node_type`-typed nodes?

Reply:
> How about "If the graph has multiple node types, then g.ndata[feat][node_type] gives the features associated with key `feat` for `node_type`-typed nodes?"

This one is better.